pkg/cue/process/handle_test.go

```diff
@@ -103,9 +103,21 @@ image: "myserver"
 		WorkflowName:    "myworkflow",
 		PublishVersion:  "mypublishversion",
 	})
-	ctx.SetBase(base)
```

Comment:
We do not need to handle so many error in unit test as they will not be triggered.